website/content/projects/2019/spec.md

```diff
@@ -403,9 +404,54 @@ You are welcome to create pull requests that do not affect the master branch, an
+
+For example, you could provide prompts like the following (the first two are examples of relatively high-level prompts, while the latter three are more specific):
+
+1. I feel like I'm using Slack and GitHub effectively, but I wonder if there are any ways in which I could be using them more effectively.
+2. I always procrastinate on my tasks until the 1-2 days before they are due, and I'd like to make more steady progress throughout the sprint. Do you have any suggestions on how I could do this?
+3. Sometimes I feel like I go off and work on one of my tasks, and when I touch base with everyone else after a few days, some of the work I've done turns out to be headed in the wrong direction, and it seems like that could've been avoided if I had known what everyone else was doing. How could I make sure I stay informed of everyone else's progress?
+4. I feel like I sometimes derail conversations by getting too lost in the weeds when talking about my work, even though I may have a valuable point to make. How could I avoid doing this?
+5. I know we create tasks at the start of each sprint, and I know which ones I'm going to be working on but, if I'm working with other people on the task, sometimes I'm not sure what exactly I should be working on. How can I make sure this doesn't happen?
+ 
+After you convey these prompts to the students who will be giving you feedback, they must convey that feedback to you. Please give your feedback givers at least one day to think about your prompts; after that, we encourage you to meet in person (ideally with both the feedback givers at the same time) to discuss your feedback (if meeting in person is hard to arrange, they can also convey their feedback over Slack, e-mail, etc.)
+
+In the end, you must write a brief report (at least 3-4 paragraphs, and ideally no more than 700 words) for each of the two pieces of feedback you provide (each to a separate student). This report must include the following:
+
+- The prompts you were given
+- How the feedback was conveyed. If you met in person, please specify when, for how long, and whether it was a one-on-one meeting, or whether the other feedback giver was there.
+- A summary of the feedback you conveyed.
+- If there was any follow-up (e.g., if the feedback recipient acted on some of your feedback, and told you the outcome), make sure to mention it in your report.
```

Comment:
Excellent. As you know I'm a big fan of following some steps upon the receipt of feedback, so it's great that they'll get some practice doing that.